### ISSUE TITLE
Archmage Leryda

### ISSUE THREAD

**Issue by Mibu86** [CoronaCore/Issues#33]
The quest giver outside Kara ...even if ure exalted with the Violet Eye... u try to talk to her for the Quest that rewards those rep rings... no gossip..shes frozzen..u cant get the quest out of her
(mangos one)

**Comment by Mibu86** [comment]
FIx   .npc flag 2 instead of 3... but change it from creature_template to be permanent

**Comment by Salja** [comment]
I will check this

**Comment by Mibu86** [comment]
believe me it work's just change flag to 2 and move to fixxed

**Comment by Salja** [comment]
Hmm the problem is sniff from blizz sayed flag 3 so must be other issues with it so must more check what is worng

**Comment by Salja** [comment]
Example worlk database use flag 3 too and works on wotlk

**Comment by Salja** [comment]
I think missing gossip option i check it tormorrow morning

**Comment by Mibu86** [comment]
i know the mangos moto :)) ....from what i noticed all the quests related on this npc works with flag 2 so i rlly dont care if is the correct number or not as long as it works flawlles :P

**Comment by Salja** [comment]
Yes ppls what using for playing can say wehn works with 2 than i use it xD :-) but dor devleopers must be 100% sure what is the right flag thats is not mangos moto thats is the fun of devleoping make all right wehn you can is salja moto

**Comment by Salja** [comment]
For a temp fix you can use

Update creature_template set NpcFlags=2 where entry = 18253;
